### ISSUE TITLE
Error on float term

### ISSUE THREAD

**Issue by bronzehedwick** [akinsho/nvim-toggleterm.lua#26]
First off, thanks for the great plugin!

I was excited to test the new float term functionality, but am getting an error. It's entirely possible I'm doing something wrong in my configuration. The documentation is not entirely clear.

I am setting `direction = 'float'`, which by looking at the code seems to be the way to opt in, although in the read me documentation float was not presented as an option (although `float_opts` is). I set up `float_opts` with some values and get this error when invoking the terminal:

<img width="964" alt="Screen Shot 2021-04-20 at 10 40 41 AM" src="https://user-images.githubusercontent.com/637174/115416100-734d3100-a1c5-11eb-9b04-164a1d332ba2.png">

I tried removing the `border` key, and removing `float_opts` entirely, but no dice.

Any help would be appreciated.

Thanks again!

**Comment by akinsho** [comment]
@bronzehedwick thanks for trying out the plugin 👍🏿 .

Yeah sorry the documentation is mission on the float direction key, I'll update that. As to the error you are seeing it seems your version of nvim nightly is too old. Native border support was added a few weeks ago and this functionality relies on that. Try updating your nvim and trying again. I just tested with your config locally and that appears to work fine.
